package.json

```diff
@@ -31,6 +31,9 @@
   },
   "devDependencies": {
     "react": "18.2.0",
-    "react-native": "0.71.3"
+    "react-native": "0.72.5"
+  },
+  "dependencies": {
+    "expo": "^49.0.13"
```

Comment:
Is expo is required to be part of Bluedot plugin? 
Can't it be added in the ReactMin App as we are not using any feature of Expo inside the plugin?